repository/src/main/java/com/michaldrabik/repository/images/EpisodeImagesProvider.kt

```diff
@@ -74,4 +62,14 @@ class EpisodeImagesProvider @Inject constructor(
       else -> mappers.image.fromDatabase(cachedImage).copy(type = type)
     }
   }
+
+  private fun findBestImage(images: List<TmdbImage>): TmdbImage? {
```

Comment:
I don't think that is even necessary since, in case of episodes, images never come with labels, titles etc. that would need to be localized.

Reply:
Completely agree with you, but as TMDB does allow you to filter by language, I don't know if in some strange case an episode image can be localised, that's why I put it there, as I think it doesn't add much complexity and helps us to unify styles.

Reply:
In this case I would much rather follow KISS and YAGNI rules so avoid adding additional code "just in case" :)

Reply:
With that argument nothing more to say xD 

Removed the findBest the episodes!